internal/commands/inspect_builder_test.go

```diff
@@ -406,46 +446,46 @@ Detection Order:
 			})
 
 			it("missing creator info is skipped", func() {
-				h.AssertNil(t, command.Execute())
-				h.AssertNotContains(t, outBuf.String(), "Created By:")
+				assert.Succeeds(command.Execute())
+				assert.NotContain(outBuf.String(), "Created By:")
```

Comment:
I know that you didn't add `NotContain`, but given that the pattern has been plural for asserts (`assert.Succeeds`, `assert.Contains`, `assert.ErrorContains`), I find it odd for this not to have an `s` at the end.